src/presign/participant.rs

```diff
@@ -1117,17 +1132,13 @@ impl PresignKeyShareAndInfo {
                 F_hat.clone(),
                 self.keyshare_public.X,
             ),
-            &PiAffgSecret::new(
-                self.keyshare_private.x.clone(),
-                beta_hat.clone(),
-                s_hat,
-                r_hat,
-            ),
+            &secret,
             context,
             &mut transcript,
             rng,
         )?;
         let mut transcript = Transcript::new(b"PiLogProof");
+        let secret = ProverSecret::new(sender_r1_priv.gamma.clone(), sender_r1_priv.nu.clone());
```

Comment:
thought: it's a little weird that this uses different naming conventions than the other proof types but I think that is a whole separate issue for consistency across ZKPs.

Reply:
Ah yes, good catch! I think we should stick to `CommonInput` and `ProverSecret`, it seems we started that with some of the refactors but then I didn't include those changes in the `PiAffgProof` refactor.

Reply:
Ah, gotcha. Made #361 and added an explicit note to the other proof refactors to include that change.